.github/workflows/e2e-test.yml

```diff
@@ -28,12 +28,12 @@ jobs:
     strategy:
       fail-fast:     false
       matrix:
-        woocommerce: [ '4.0.0', '4.5.0', 'beta' ]
+        woocommerce: [ '4.4.0', '5.5.0', '5.7.0', 'beta' ]
```

Comment:
Should we have `5.6.0` too?

Reply:
Not really, I'd suggest even skipping `'5.5.0'` . @vbelolapotkov what was the idea behind adding both `5.5.0` and `5.7.0` here?

Reply:
The idea was to test both edges of L-2 support range. So the full range is:
Min WC | L-2 | L(atest) | beta

Reply:
> Min WC | L-2 | L(atest) | beta

Thanks for the explanation. If so, this change makes sense.
I've added a comment to explain why we keep this, which would be helpful for the next maintainers. Ref 16148f3